nuget.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<clear />
+		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="myget" value="%MYGET_FEED%" />
```

Comment:
the reason I was using `dotnet nuget add source` was to avoid exactly this - now this nuget.config is invalid and VS wont load it:
```
---------------------------
Microsoft Visual Studio
---------------------------
NuGet operation failed

NuGet.Config is not valid XML. Path: 'D:\GitHub\WopiHost\NuGet.Config'.
  Name cannot begin with the '<' character, hexadecimal value 0x3C. Line 6, position 2.
---------------------------
OK   
---------------------------
```